cmd/crictl/main.go

```diff
@@ -140,14 +172,17 @@ func main() {
 			Name:    "runtime-endpoint",
 			Aliases: []string{"r"},
 			EnvVars: []string{"CONTAINER_RUNTIME_ENDPOINT"},
-			Value:   defaultRuntimeEndpoint,
-			Usage:   "Endpoint of CRI container runtime service",
+			Usage: "Endpoint of CRI container runtime service (default: " +
+				"uses in order first successful one of \"dockershim, " +
+				"containerd, cri-o\"). Default is now deprecated and the " +
+				"endpoint should be set instead.",
```

Comment:
not sure if I like the logical list here or the actual list...
{"unix:///var/run/dockershim.sock", "unix:///run/containerd/containerd.sock", "unix:///run/crio/crio.sock"}

Reply:
does this also run on Windows and is this the same CLI output? The defaults were not shown before in the usage.

Reply:
ok, updated now.